fhirr4/ballerina/src/main/resources/fhirservice/http_service_builder.bal

```diff
@@ -45,6 +50,17 @@ isolated function getHttpService(Holder h, r4:ResourceAPIConfig apiConfig) retur
                     }
                     r4:FHIRContext fhirContext = check r4:getFHIRContext(ctx);
                     string id = paths[paths.length() - 1];
+                    string resourceType = split[split.length() - 2];
```

Comment:
shall we rename the var to something more intuitive. i.e: requestPathParts

Reply:
done